RecoTracker/CkfPattern/python/CkfTrackCandidatesP5_cff.py

```diff
@@ -15,9 +15,9 @@
 import copy
```

Comment:
Error occur when I remove this copy here : 
```
 File "/afs/cern.ch/work/j/jelee/ServiceWork/EventContent/CMSSW_11_2_X_2020-07-20-2300/python/RecoMuon/MuonIdentification/cosmics_id.py", line 12, in <module>
    cosmicsVetoTrackCandidates = copy.deepcopy(ckfTrackCandidatesP5)
NameError: name 'copy' is not defined
```
https://github.com/cms-sw/cmssw/blob/5d182778be59af8a26000b207fdafcc22cc89e2e/RecoMuon/MuonIdentification/python/cosmics_id.py#L11
from RecoTracker.CkfPattern.CkfTrackCandidatesP5_cff import *

(I will remove this copy when I update the RecoMuon)

Reply:
Updating RecoMuon/MuonIdentification/python/cosmics_id.py seems to be rather straightforward: I think you should do it right now, for that specific file, so that you won't forget to clean up this one later on.